modules/cnv-about-upgrading-cnv.adoc

```diff
@@ -10,8 +10,7 @@ receive updates as they become available.
 
 .Additional information
```

Comment:
I'd probably change this heading to a statement that introduces the list. (And if there's no good way to do that, I'd break up the list in such a way that you could.)